Formula/v2ray.rb

```diff
@@ -0,0 +1,49 @@
+class V2ray < Formula
+  desc "Platform for building proxies to bypass network restrictions"
+  homepage "https://v2fly.org/"
+  url "https://github.com/v2fly/v2ray-core/archive/v4.32.1.tar.gz"
+  sha256 "f5553ffd04ff226573e8132d9beaa63f4d8f4882eba047225b33620848bc6917"
+  license all_of: ["MIT", "CC-BY-SA-4.0"]
+  head "https://github.com/v2fly/v2ray-core.git"
+
+  depends_on "go" => :build
+
+  resource "geoip" do
+    url "https://github.com/v2fly/geoip/releases/download/202011150541/geoip.dat"
+    sha256 "11b7c3bfc5715c42d26b0e4bcf51d38c157eae9ab4b9e8391d702681e385dbcd"
+  end
+
+  resource "geosite" do
+    url "https://github.com/v2fly/domain-list-community/releases/download/20201117215431/dlc.dat"
+    sha256 "32ba60cb90c9f6951e2de12bf3a8e9fd5fbccee4d1706421f1a9413303e75a2e"
+  end
+
+  def install
+    ldflags = "-s -w -buildid="
+    system "go", "build", *std_go_args,
+                 "-ldflags", ldflags,
+                 "./main"
+    system "go", "build", *std_go_args,
+                 "-ldflags", ldflags,
+                 "-tags", "confonly",
+                 "-o", bin/"v2ctl",
+                 "./infra/control/main"
+
+    (etc/"v2ray").install "release/config/config.json" => "config.json"
+
+    resource("geoip").stage do
+      (pkgshare/"v2ray").install "geoip.dat"
```

Comment:
How come it still works when you changed the location? Aren't these required to run v2ray?

Reply:
Sorry for the confusion, the correct location of the resources is `pkgshare`: https://github.com/v2fly/v2ray-core/blob/9bf09b2242e564bf9ecadd3812e8342d5f2a4282/common/platform/others.go#L31-L32
These resources are used if certain routing rules are defined in the config. The test uses a minimal config so the test passed even though the resources are placed in the wrong location.

Reply:
In that case we should expand the test to cover this.